corehq/apps/sso/tests/test_message_helpers.py

```diff
@@ -0,0 +1,29 @@
+from unittest import mock
+
+from django.test import SimpleTestCase, RequestFactory
+
+from corehq.apps.sso.utils.message_helpers import show_sso_login_success_or_error_messages
+
+
+class TestShowSsoLoginSuccessOrErrorMessages(SimpleTestCase):
```

Comment:
The data structure used here doesn't prohibit a request having both success and error messages. Is this a case that can occur? If so, what happens, and should a test document that?

Reply:
yes, that is possible. added a test case for that, thanks!